### ISSUE TITLE
NaN loss and gradients when training the neural network

### ISSUE THREAD

**Issue by Teja10** [owensgroup/rtx_nerf#11]
Currently getting nan values for gradients leading to nan loss. The issue is likely something to do with loss scaling
